Mathlib/NumberTheory/Cyclotomic/PrimitiveRoots.lean

```diff
@@ -156,7 +156,6 @@ noncomputable def embeddingsEquivPrimitiveRoots (C : Type*) [CommRing C] [IsDoma
       left_inv := fun _ => Subtype.ext rfl
       right_inv := fun _ => Subtype.ext rfl }
 
--- Porting note: renamed argument `φ`: "expected '_' or identifier"
```

Comment:
What's going on here? Does `φ` still not work? If so, then it seems this should still be tracked. If not, then why can't we make that change in the lemma?

Reply:
Oh, I didn't even see the prime. Reverted.